### PR TITLE
Feature/graph/serve static

### PR DESCRIPTION
**Pull request by aguua**
folder given under var 'CLI_STATIC_CONTENT' should include  **index.html** file

### CODE REVIEW

File: graph/Makefile [hunk 1]
```diff
@@ -1,10 +1,12 @@
 .DEFAULT_GOAL := help
+export UI_STATIC_CONTENT = $(PWD)/tmp
+
 # Porcelain
 # ###############
 .PHONY: env-up env-down env-recreate container run build lint test
 
 run: setup ## run the app
-	python src/main.py <./tmp/fake_graph.json
+	python src/main.py $(UI_STATIC_CONTENT)
```
Comment (allgreed) [inline graph/Makefile:9]: I've talked about an environment variable - but if you have good reasons for doing it this way we can absolutely go with it. Do you?

File: graph/src/main.py [hunk 2]
```diff
@@ -15,22 +15,25 @@
 @app.route('/data')
 @cross_origin()
 def data_index():
-    my_output = json.loads(input_data)
+    content = get_json_content(os.path.join(ui_static_content, "fake_graph.json"))
```
Comment (allgreed) [inline graph/src/main.py:18]: how about keeping the original mechanism and piping the data via additional Make target? Or using default data when no data is provided?

File: graph/src/main.py [hunk 3]
```diff
@@ -15,22 +15,25 @@
...
         schema=get_json_content("../contracts/graph-to-ui.json")
     )
     return {'my data json': my_output}
 
-# TODO: serve static content on /
+
 @app.route('/')
 def index():
-    return {"Status": "ok"}
+    return render_template("index.html", path_to_static=ui_static_content)
```
Comment (allgreed) [inline graph/src/main.py:30]: nit: I'm not supper familiar with Flask - do we need `render_template`? If that's their way of saying "serve file" I'm cool with that, just asking ;)

**Review by allgreed — CHANGES_REQUESTED** [review]
Minor stuff and we're good to merge :D

File: graph/src/main.py (hunk 3 above)
Comment (aguua) [reply graph/src/main.py:30]: That's one way to say 'serve file', there is also something like "send_static_file" but it doesn't work xD I can look at it later.

File: graph/Makefile (hunk 1 above)
Comment (aguua) [reply graph/Makefile:9]: no xD

**Review by allgreed — APPROVED** [review]

**Pull request merged**